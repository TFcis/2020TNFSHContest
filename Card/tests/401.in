100
78 19 91 87 66 95 18 85 34 69 1 94 8 99 47 64 5 76 98 23 90 12 25 81 57 65 2 41 17 84 42 77 54 24 45 39 21 73 83 27 53 61 20 96 67 26 28 79 75 44 60 100 11 30 16 58 33 13 14 35 80 4 89 22 32 48 46 15 10 92 49 36 74 29 59 70 50 9 52 51 86 68 93 62 63 40 82 37 56 71 72 55 88 31 38 43 6 3 97 7
31 91 9 43 100 65 73 49 58 46 48 1 33 97 45 21 2 72 32 94 20 44 61 99 84 75 38 85 90 29 95 7 87 54 80 25 68 16 3 15 13 53 81 69 55 22 63 64 50 57 62 12 40 30 4 59 27 26 10 92 56 23 86 42 89 74 41 28 70 78 19 82 8 67 18 66 51 96 39 60 36 14 52 34 37 71 76 24 17 77 98 88 79 83 35 6 5 93 11 47
879786296904251838
