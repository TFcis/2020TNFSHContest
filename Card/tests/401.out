92 45 97 36 87 94 84 71 1 2 88 10 82 67 65 58 38 11 24 69 14 64 85 72 66 22 25 21 49 40 9 30 70 28 95 6 41 80 81 13 54 91 68 90 59 73 86 52 50 60 18 96 34 89 76 44 4 79 33 56 75 74 39 83 78 46 8 63 23 31 62 27 17 53 61 29 77 93 37 32 99 47 12 7 15 57 48 19 42 98 51 5 55 26 43 100 16 35 3 20
